crates/holochain_conductor_api/src/admin_interface.rs

```diff
@@ -392,6 +392,10 @@ pub enum AdminRequest {
     DeleteCloneCell(Box<DeleteCloneCellPayload>),
 
     /// Info about storage used by apps
+    ///
+    /// # Returns
+    ///
+    /// [`AdminResponse::StorageInfo`]
```

Comment:
The other requests all document their return type to make these enums easier to navigate. Updating for consistency